src/Dfe.PlanTech.DatabaseUpgrader/Scripts/2024/20241010_0545_FixRichTextSlugView.sql

```diff
@@ -0,0 +1,49 @@
+ALTER VIEW [Contentful].[RichTextContentsBySlug] AS
+    WITH RichTexts AS (
+        SELECT
+            [RichTextId],
+            ISNULL([Warning].[Id], [TB].[Id]) AS ContentId
+        FROM
+            [Contentful].[TextBodies] AS TB
+            LEFT JOIN [Contentful].Warnings AS Warning ON [Warning].[TextId] = [TB].[Id]
```

Comment:
Thinking out loud, I'm wondering why we have to do this, and left join to warnings, after all, it just contains a text body, which ought to be connected to the page, so would come up in this query anyway?? 🤔 

will have a look at the counter part to this one